octokit/issues.go

```diff
@@ -1,49 +1,80 @@
 package octokit
 
 import (
-	"net/url"
-	"time"
-
 	"github.com/jingweno/go-sawyer/hypermedia"
+	"time"
 )
 
 // RepoIssuesURL is a template for accessing issues in a particular
 // repository for a particular owner that can be expanded to a full address.
-var RepoIssuesURL = Hyperlink("repos/{owner}/{repo}/issues{/number}")
+var RepoIssuesURL = Hyperlink("repos/{owner}/{repo}/issues{/number}?" +
+	"filter={filter}&state={state}&labels={labels}&sort={sort}&" +
```

Comment:
@dhruvsinghal I think @pengwynn meant for you to update the line (the regex) to what he commented haha